torch/csrc/jit/backends/backend_detail.cpp

```diff
@@ -293,7 +293,8 @@ Module codegen_backend_module(
         // backend_execute.
         TORCH_INTERNAL_ASSERT(default_value.has_value());
         std::stringstream def_ss, fwd_ss;
-        def_ss << name << "=";
+        // Annotate type of the arg
+        def_ss << name << ":" << arg.type()->annotation_str(nullptr) << "=";
```

Comment:
```suggestion
        def_ss << name << ": " << arg.type()->annotation_str() << "=";
```
Not that anyone will be looking at this code but there is usually a space after the `:`

Reply:
Yeah makes sense. Will make that change.